packages/schema/package.json

```diff
@@ -34,6 +34,7 @@
     "fs-extra": "^9.0.0",
     "istanbul": "0.4.5",
     "markdown-toc": "^1",
+    "mocha": "^7",
```

Comment:
This was apparently needed to run schema-only tests? Since it looks in `schema/node_modules/.bin` instead of `root/node_modules`

Reply:
I don't think this is necessary. When we run `npm test` or `yarn test` under `packages/schema`. `npm` or `yarn` is smart enough to find `mocha` at the root level.

Reply:
Ah, `npm test` was actually my issue:

```
% npm test

> zapier-platform-schema@10.2.0 test /Users/david/projects/zapier/platform/packages/schema
> mocha -t 10000 --recursive test

sh: mocha: command not found
npm ERR! Test failed.  See above for more details.
```

`yarn test` works as expected though. I'll remove!

Reply:
Oh yeah, back to think about it, it only works on `yarn test` works because of yarn workspaces.